ops/ansible/playbooks-ccs/data_server_collect_flame_graph.yml

```diff
@@ -5,17 +5,17 @@
 #
 # Usage:
 #
-#     $ pipenv run ./ansible-playbook-wrapper data_server_collect_flame_graph.yml --extra-vars "perf_record_seconds=15 find_process_command='pgrep java' jdk_path=/etc/alternatives/java_sdk_1.8.0_openjdk" --limit bluebutton-healthapt-test-data-server-a-1
+#     $ pipenv run ./ansible-playbook-wrapper data_server_collect_flame_graph.yml --inventory='somehost.ec2.internal' --extra-vars "perf_record_seconds=15 find_process_command='pgrep -f bfd-server-launcher.*jetty' java_user='bfd' jdk_path=/etc/alternatives/java_sdk_1.8.0_openjdk"
```

Comment:
Fun fact: `pgrep -f 'java.*DataServerLauncherApp'` would be a much better filter/search command here, but won't work for reasons that escape me. It looks like maybe `pgrep -f` won't search the command line past a certain character limit?